funnel/assets/js/utils/helper.js

```diff
@@ -402,7 +402,7 @@ const Utils = {
     }
     return false;
   },
-  getFaviconHTML(icon, iconSize = 'body', baseline = true, cssClassArray=[]) {
+  getFaviconHTML(icon, iconSize = 'body', baseline = true, cssClassArray = []) {
```

Comment:
Thought this was renamed already. Looks like this needs to be updated against `main`.